console.json

```diff
@@ -4,8 +4,8 @@
   "description": "No effort, universal logger",
   "license": "MIT",
   "esy": {
-    "build": "dune build -p console",
-    "install": "esy-installer console.install"
+    "build": "esydune build",
+    "install": "esydune install"
```

Comment:
Should we make `esy`'s built in install process automatically look to see if there's a `.install` file that matches your package name (minus any scope) and just use that one so you don't have to specify it or use `esydune install`?

Reply:
Good idea! Will implement now.

Reply:
PR updated not to include redundant `"install"` config.